src/styles.scss

```diff
@@ -29,6 +29,10 @@ html {
 	min-height: 100%;
 }
 
+article, aside, footer, header, nav, section {
```

Comment:
Not sure this is needed.  bootstrap styles that are imported above includes the follow:

```css
article, aside, figcaption, figure, footer, header, hgroup, main, nav, section {
  display: block;
}
```